### PR TITLE
Change return format of images with properties

### PR DESCRIPTION
**Pull request by vasokot**
JetBrains change format for images with proprieties at YouTrack, so we need to add some changes at jira2markdown

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Restore virtualenv from cache
         id: cache
-        uses: actions/cache@v1
+        uses: actions/cache@v3
```
Comment (catcombo) [inline .github/workflows/ci.yml:26]: Could you please also bump other actions version?

- actions/checkout@v2 -> actions/checkout@v4
- actions/setup-python@v2 -> actions/setup-python@v5

Could you please move these changes to a separate commit with the following comment: `chore(ci): bump actions version`

File: jira2markdown/markup/images.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def action(self, tokens: ParseResults) -> str:
             if name in self.ALLOWED_ATTRS
         )
         if attrs_str:
-            return f'<img src="{tokens.url}" {attrs_str} />'
+            return f"![{tokens.url}]({tokens.url})" + "{" + attrs_str + "}"
```
Comment (catcombo) [inline jira2markdown/markup/images.py:35]: `attrs_str` should not contain a value in quotes. I just tested image attributes in the YouTrack. The following image has been render with the correct image size:
```markdown
![Markdown logo](https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/208px-Markdown-mark.svg.png "Markdown"){width=100}
```
But if attribute value is surrounded with quote, then image size doesn't change
```markdon
![Markdown logo](https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/208px-Markdown-mark.svg.png "Markdown"){width="100"}
```
You can change this behavior by replacing `f'{name}="{value}"'` a few lines above with the `f"{name}={value}"`. Related tests should be updated as well.

**Review by catcombo — CHANGES_REQUESTED** [review]
Changes looks good and well prepared! Thank you! I have only two additional suggestions.

1. Please update images section in the `README.md` to reflect changes. Commit these changes with the following commit message `docs(readme): use YouTrack format to render image with attributes`.
2. Change commit message for the code changes to `feat(images): use YouTrack format to render image with attributes` to follow https://www.conventionalcommits.org/en/v1.0.0/

**Comment by vasokot** [comment]
All changes applied at https://github.com/catcombo/jira2markdown/pull/37

**Pull request closed without merging**